custom_components/ai_thermostat/climate.py

```diff
@@ -64,7 +64,9 @@
 
 ATTR_STATE_WINDOW_OPEN = "window_open"
 ATTR_STATE_NIGHT_MODE = "night_mode"
-ATTR_STATE_CALL_FOR_HEAT = "summer"
+ATTR_STATE_SUMMER = "summer"
+ATTR_STATE_LAST_CHANGE = "last_change"
+ATTR_STATE_DAY_TEMP = "last_day_temp"
```

Comment:
Shouldn't this be called setpoint? I mean, it's a temperature, but calling it temperature feels like a sensor value, not a user setting.